core/import_data/import_progress_reports.py

```diff
@@ -0,0 +1,124 @@
+import csv
+import logging
+
+from django.conf import settings
+
+from core.import_data.utils import get_object_by_code
+from core.import_data.utils import get_object_by_name
+from core.import_data.utils import parse_date
+from core.models import Agency
+from core.models import Country
+from core.models import Project
+from core.models import Region
+from core.models.project import ProjectProgressReport
+from core.models.project import ProjectSector
+from core.models.project import ProjectStatus
+from core.models.project import ProjectType
+
+logger = logging.getLogger(__name__)
+
+
+def import_progress_reports():
+    logger.info("⏳ importing progress reports")
+    file_path = settings.IMPORT_DATA_DIR / "progress_report" / "tbProgress.csv"
+    with file_path.open("r") as csvfile:
+        reader = csv.DictReader(csvfile)
+        for index, item in enumerate(reader):
+            project = get_object_by_code(Project, item["code"], "code", index, logger)
+            country = get_object_by_code(
```

Comment:
the code from tbProgress is not the same with our iso3 (from the database). You should use the table tbCountryId (nextcloud) to find the country name and after that you should search by name (you can use get_country_by_name from utils)

Reply:
Sounds good!